lib/Mojo/Server/Daemon.pm

```diff
@@ -85,8 +85,14 @@ sub _build_tx {
   my $tx = $self->build_tx->connection($id);
   $tx->res->headers->server('Mojolicious (Perl)');
   my $handle = $self->ioloop->stream($id)->handle;
-  $tx->local_address($handle->sockhost)->local_port($handle->sockport);
-  $tx->remote_address($handle->peerhost)->remote_port($handle->peerport);
+  if ($handle->isa('IO::Socket::UNIX')) {
+    $tx->local_address($handle->hostpath)->local_port(-1);
+    $tx->remote_address($handle->peerpath)->remote_port(-1);
```

Comment:
Yes, the issue here is that those slots doesn't make sense for UNIX sockets, so, If you aim for correctness, probably, setting those to `undef` is the right thing to do.

After a shallow Inspection of the Mojo code, it seems to me that they are not used by Mojo internally.